### DISCUSSION TITLE
Update to Bitbucket RSA keys

### DISCUSSION THREAD

**Discussion by dbcelm** [argoproj/argo-helm#2151] [General]
Hi Team, can you please update BitBucket RSA key within default values in next minor release:

https://bitbucket.org/blog/ssh-host-key-changes

**Comment by jmeridth** [comment]
This was handled with https://github.com/argoproj/argo-helm/pull/2062 for Argo-cd chart. Which chart specifically? What version? Need more details. 

Requests like this would be an issue, not a discussion. For future reference. Thank you.
